app/scripts/services/sql.js

```diff
@@ -12,6 +12,9 @@ angular.module('sql', [])
       return obj;
     };
     return function(sqlQuery, headers) {
```

Comment:
suggestion:

```js
  .factory('queryResultToObjects', function() {
    var toObject = function(cols, row) {
      var obj = {};
      if (cols.length === row.length) {
        for (var i = 0; i < cols.length; i++) {
          obj[cols[i]] = row[i];
        }
      }
      return obj;
    };
    return function(sqlQuery) {
      return sqlQuery.rows.map(function(o) {
        return toObject(sqlQuery.cols, o);
      });
    };
})
```

this however would require to name the select list items correctly.